MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,308 @@
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName" || e.DataFormItem.FieldName == "MiddleName" || e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.GroupName = "Name";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing editors order in the group
+
+The order of the editors in the group can be changed by using attributes. You can set the order of data form items in group by using the `Order` property along with `GroupName` property in display attribute.
+
+{% tabs %}
+{% highlight c# %}
+public class ContactInfo
+{
+    [Display(GroupName = "Name", Order = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name", Order = 1)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name", Order = 2)]
+    public string MiddleName { get; set; }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The order of the `DataFormGroupItem` can also be changed by handling the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+        private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.GroupName == "Name")
+            {
+                if (e.DataFormItem.FieldName == "FirstName")
+                {
+                    e.DataFormItem.RowOrder = 0;
+                }
+                else if (e.DataFormItem.FieldName == "LastName")
+                {
+                    e.DataFormItem.RowOrder = 2;
+                }
+                else if (e.DataFormItem.FieldName == "MiddleName")
+                {
+                    e.DataFormItem.RowOrder = 1;
+                }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Changing group name
+
+The `GroupName` for the group can be changed in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+       private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem != null)
+            {
+                if (e.DataFormItem.GroupName == "Name")
+                {
+                    e.DataFormItem.GroupName = "Name Group";
+                }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the layout for the group
+
+The linear or grid layout for the particular group can be loaded by handling the `GenerateDataFormItem` event.
+By setting the `ColumnCount` property in the data form, non-grouped items only will be arranged in the grid layout. To load the grid layout, set the [ColumnCount] for the `DataFormGroupItem`
+
+{% tabs %}
+{% highlight c# %}
```

Comment:
Add ContactModel class with this example

Reply:
Added ContactModel class